<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>progressDialog</class>
 <widget class="QDialog" name="progressDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>521</width>
    <height>203</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Build Progress</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>501</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="lblStreams">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>15</y>
      <width>471</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Stream: 0/0</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="streamProgress">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>451</width>
      <height>16</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
    <property name="textDirection">
     <enum>QProgressBar::TopToBottom</enum>
    </property>
   </widget>
   <widget class="QLabel" name="lblCurrent">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>471</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Current Stream: 0/0</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="currentProgress">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>451</width>
      <height>16</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
    <property name="textDirection">
     <enum>QProgressBar::TopToBottom</enum>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>170</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Cancel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
